Dockerfile

```diff
@@ -53,6 +53,9 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     jo=1.9-1 \
     jq=1.6-2.1 \
     netcat-traditional=1.10-47 \
+    iputils-ping libpcap0.8 \
+    mitmproxy sudo \
+    iproute2 \
```

Comment:
iputils-ping libpcap0.8 ... required for knocked.
mitmproxy sudo ... for communication capture.
iproute2 ... To check the interface with the ip command..